pennathletics/__init__.py

```diff
@@ -1,5 +1,5 @@
 __title__ = 'pennathletics'
 __version__ = '0.0.1'
-__author__ = 'Adel Qalieh'
+__author__ = 'Adel Qalieh, Andrew Fischer'
```

Comment:
We should get rid of this and replace with `Penn Labs` and have a separate `AUTHORS.rst` in the vein of [requests](https://github.com/kennethreitz/requests/blob/master/AUTHORS.rst)